.bumpversion.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 0.1.8
+current_version = 0.1.9
```

Comment:
@karimkawambwa right now bumpversion should only occur in master branch directly.

Reply:
This is to avoid conflicts and for it to happen just before / as part of publish.